File: lalalala/pipelines/lineageRegression/code/src/main/scala/io/prophecy/pipelines/lineageregression/graph/Reformat_3.scala
package io.prophecy.pipelines.lineageregression.graph

import io.prophecy.libs._
import io.prophecy.pipelines.lineageregression.udfs.UDFs._
import io.prophecy.pipelines.lineageregression.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_3 { def apply(context: Context, in: DataFrame): DataFrame = in }

File: MigrationPy/pipelines/p1/code/p1/graph/Subgraph_2/Reformat_3.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from p1.udfs.UDFs import *

def Reformat_3(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0
